lib/config/common-media-library.api.md

```diff
@@ -627,7 +627,8 @@ export function validateTracks(tracks: Track[]): TrackValidity;
 export type VideoTrack = Track & {
     width: number;
     height: number;
-    frameRate: string;
+    frameRateNum: number;
+    frameRateDen?: number;
```

Comment:
I would not save characters in the names of the attributes. So that there is no guessing and nor confusion for whoever that reads the code.
```suggestion
    frameRateNumerator: number;
    frameRateDenominator?: number;
```